src/cim/components/featureSupportLevels/featureStateUtils.tsx

```diff
@@ -1,28 +1,53 @@
-import { FeatureId, SupportLevel } from '../../../common/types';
-import { Cluster } from '../../../common/api/types';
+import { FeatureId, OpenshiftVersionOptionType, SupportLevel } from '../../../common/types';
 import { TFunction } from 'i18next';
 
-const clusterExistsReason = 'This option is not editable after the draft cluster is created';
+const getClusterExistsReason = (t: TFunction) =>
+  t('ai:This option is not editable after the draft cluster is created');
 
-const getSNODisabledReason = (cluster: Cluster | undefined, isSupported: boolean, t: TFunction) => {
-  if (cluster) {
-    return clusterExistsReason;
+const getSNODisabledReason = (clusterExists: boolean, isSupported: boolean, t: TFunction) => {
+  if (clusterExists) {
+    return getClusterExistsReason(t);
   }
   if (!isSupported) {
     return t('ai:Single-Node OpenShift is not supported in this OpenShift version');
   }
-  return '';
+  return undefined;
+};
+
+const getArmDisabledReason = (
+  clusterExists: boolean,
+  versionName: string,
+  versionOptions: OpenshiftVersionOptionType[],
+  isSupported: boolean,
+  t: TFunction,
+) => {
+  if (clusterExists) {
+    return getClusterExistsReason(t);
+  }
+  if (!isSupported) {
+    return t('ai:arm64 is not supported in this OpenShift version');
+  }
+  // TODO(jtomasek): versionOptions derived from clusterimagesets need to include cpuArchitectures array to correctly determine this
```

Comment:
Is that blocked on something recently?

Reply:
The `clusterimageset` resource does not have any property which describes the image architecture. Only lead is the image name/url but that is very unreliable and fragile.